### DISCUSSION TITLE
MIMIC-IV Admission Type Meanings

### DISCUSSION THREAD

**Discussion by larabg** [MIT-LCP/mimic-code#1712] [MIMIC-IV]
Hi there. 
Can anyone provide insight into what the admission types refer to? Specifically, what do EU OBSERVATION and EW EMER mean?
Thanks!

**Comment by marymlucas** [comment]
Hi @larabg you may find some answers here - [https://github.com/MIT-LCP/mimic-code/discussions/1215](https://github.com/MIT-LCP/mimic-code/discussions/1215)
